src/main/java/com/researchspace/netfiles/NfsFactory.java

```diff
@@ -89,10 +89,16 @@ public NfsClient getNfsClient(String nfsusername, String nfspassword, NfsFileSys
           fileSystem.getClientOption(NfsFileSystemOption.SFTP_SERVER_PUBLIC_KEY));
     }
     if (NfsClientType.IRODS.equals(clientType)) {
+      int irodsPort;
+      if (StringUtils.isEmpty(fileSystem.getClientOption(NfsFileSystemOption.IRODS_PORT))) {
+        irodsPort = 1247;
```

Comment:
Could you possibly make `1247` a constant? maybe something lik `IRODS_DEFAULT_PORT`?